matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/DefaultSession.kt

```diff
@@ -153,10 +158,13 @@ internal class DefaultSession @Inject constructor(
     override val isOpenable: Boolean
         get() = sessionParamsStore.get(sessionId)?.isTokenValid ?: false
 
+    private var sessionScope: CoroutineScope? = null
+
     @MainThread
     override fun open() {
         assert(!isOpen)
         isOpen = true
+        sessionScope = CoroutineScope(SupervisorJob())
```

Comment:
Use a holder shared with Dagger SessionScope, so we can use it internally?

Reply:
Done